go.sum

```diff
@@ -0,0 +1,2 @@
+github.com/stretchr/testify v1.6.1 h1:hDPOHmpOpP40lSULcqw7IrRb/u7w6RpDC9399XyoNd0=
```

Comment:
@gyanesh-m is testify a production dependency?

Reply:
@captn3m0 Its need for testing, not for production. Do you want to have a separate go.mod file for prod and test ?